### ISSUE TITLE
False positive unused import in thread macro

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#796]
Repro:

```
 $ clj-kondo --lint - <<< 'clj -A:clj-kondo --lint - <<< '(ns foo (:import [foo Bar])) (defn foo [x] x) (defn bar [x] (-> x ^Bar (.execute)))''
<stdin>:1:23: warning: Unused import Bar
linting took 42ms, errors: 0, warnings: 1
```
